charts/ceph-csi-cephfs/values.yaml

```diff
@@ -206,18 +206,6 @@ provisioner:
 # selinux-enabled filesystems
 selinuxMount: true
 
-topology:
-  # Specifies whether topology based provisioning support should
-  # be exposed by CSI
-  enabled: false
-  # domainLabels define which node labels to use as domains
-  # for CSI nodeplugins to advertise their domains
-  # NOTE: the value here serves as an example and needs to be
-  # updated with node labels that define domains of interest
-  domainLabels:
-    - failure-domain/region
-    - failure-domain/zone
```

Comment:
Please update Readme also as these values are getting removed.

Reply:
@Madhu-1 where exactly you refer

Reply:
@humblec here https://github.com/ceph/ceph-csi/tree/devel/charts/ceph-csi-cephfs#configuration

Reply:
@Madhu-1 updated.thanks